<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>StreamMenu</class>
    <widget class="QMenu" name="streamMenu">
        <property name="toolTipsVisible">
            <bool>true</bool>
        </property>
        <addaction name="clearFilterAction"/>
        <addaction name="separator"/>
        <addaction name="greaterFilterAction"/>
        <addaction name="smallerFilterAction"/>
        <addaction name="matchFilterAction"/>
        <addaction name="bytesFilterAction"/>
        <addaction name="separator"/>
        <widget class="QWidgetAction" name="listFilterAction"/>
        <widget class="QWidget" name="optionsWidget">
            <property name="visible">
                <bool>false</bool>
            </property>
            <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="optionsLayout">
                <property name="spacing">
                    <number>5</number>
                </property>
                <property name="leftMargin">
                    <number>5</number>
                </property>
                <property name="topMargin">
                    <number>5</number>
                </property>
                <property name="rightMargin">
                    <number>5</number>
                </property>
                <property name="bottomMargin">
                    <number>5</number>
                </property>
                <item>
                    <widget class="QListView" name="optionsView">
                        <property name="sizePolicy">
                            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                            </sizepolicy>
                        </property>
                        <property name="maximumSize">
                            <size>
                                <width>180</width>
                                <height>120</height>
                            </size>
                        </property>
                    </widget>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="buttonsLayout">
                        <property name="spacing">
                            <number>5</number>
                        </property>
                        <item>
                            <widget class="QPushButton" name="confirmButton">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                                    </sizepolicy>
                                </property>
                                <property name="minimumSize">
                                    <size>
                                        <width>0</width>
                                        <height>32</height>
                                    </size>
                                </property>
                                <property name="text">
                                    <string>Ok</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="cancelButton">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                                    </sizepolicy>
                                </property>
                                <property name="minimumSize">
                                    <size>
                                        <width>0</width>
                                        <height>32</height>
                                    </size>
                                </property>
                                <property name="text">
                                    <string>Cancel</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
            </layout>
        </widget>
        <action name="clearFilterAction">
            <property name="enabled">
                <bool>false</bool>
            </property>
            <property name="checkable">
                <bool>false</bool>
            </property>
            <property name="checked">
                <bool>false</bool>
            </property>
            <property name="icon">
                <iconset theme="filter-off">
                </iconset>
            </property>
            <property name="text">
                <string>Clear Filter</string>
            </property>
            <property name="toolTip">
                <string>Clear all filters for this column</string>
            </property>
        </action>
        <action name="greaterFilterAction">
            <property name="enabled">
                <bool>true</bool>
            </property>
            <property name="visible">
                <bool>false</bool>
            </property>
            <property name="checkable">
                <bool>false</bool>
            </property>
            <property name="icon">
                <iconset theme="math-greater">
                </iconset>
            </property>
            <property name="text">
                <string>Greater than...</string>
            </property>
            <property name="toolTip">
                <string>Filter rows with content greater than given value</string>
            </property>
        </action>
        <action name="smallerFilterAction">
            <property name="enabled">
                <bool>true</bool>
            </property>
            <property name="visible">
                <bool>false</bool>
            </property>
            <property name="checkable">
                <bool>false</bool>
            </property>
            <property name="icon">
                <iconset theme="math-lower">
                </iconset>
            </property>
            <property name="text">
                <string>Smaller than...</string>
            </property>
            <property name="toolTip">
                <string>Filter rows with content smaller than given value</string>
            </property>
        </action>
        <action name="matchFilterAction">
            <property name="enabled">
                <bool>true</bool>
            </property>
            <property name="visible">
                <bool>false</bool>
            </property>
            <property name="checkable">
                <bool>false</bool>
            </property>
            <property name="icon">
                <iconset theme="match-regex">
                </iconset>
            </property>
            <property name="text">
                <string>Match expression</string>
            </property>
            <property name="toolTip">
                <string>Filter rows with matches the regular expression</string>
            </property>
        </action>
        <action name="bytesFilterAction">
            <property name="enabled">
                <bool>true</bool>
            </property>
            <property name="visible">
                <bool>false</bool>
            </property>
            <property name="checkable">
                <bool>false</bool>
            </property>
            <property name="icon">
                <iconset theme="match-bytes">
                </iconset>
            </property>
            <property name="text">
                <string>Match bytes</string>
            </property>
            <property name="toolTip">
                <string>Filter rows with matches the byte values</string>
            </property>
        </action>
    </widget>
    <connections>
        <connection>
            <sender>confirmButton</sender>
            <signal>clicked()</signal>
            <receiver>listFilterAction</receiver>
            <slot>trigger()</slot>
        </connection>
        <connection>
            <sender>cancelButton</sender>
            <signal>clicked()</signal>
            <receiver>listFilterAction</receiver>
            <slot>trigger()</slot>
        </connection>
    </connections>
</ui>
